contribute/style-guides/documentation-style-guide.md

```diff
@@ -54,6 +54,7 @@ For all items not covered in this guide, refer to the [Microsoft Style Guide](ht
 ### Capitalization
 
 * Grafana, Loki, and Prometheus are always capitalized unless part of a code block.
+* API names are always Title Case, and followed by "API" — for example, "Dashboard Permissions API"
```

Comment:
```suggestion
* API names are always Title Case, followed by "API"—for example, "Dashboard Permissions API"
```

Reply:
I think thats committed now, anything else i need to do?